pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -7092,6 +7115,26 @@ bool GSRendererHW::TextureCoversWithoutGapsNotEqual()
 	return false;
 }
 
+bool GSRendererHW::isDownscaleDraw(GSTextureCache::Source* src, bool no_gaps)
```

Comment:
```suggestion
bool GSRendererHW::IsDownscaleDraw(GSTextureCache::Source* src, bool no_gaps)
```

Consistent casing ;)